chall02/rgallard.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+morse = {
+	'A':'.-',
+	'B':'-...',
+	'C':'-.-.',
+	'D':'-..',
+	'E':'.',
+	'F':'..-.',
+	'G':'--.',
+	'H':'....',
+	'I':'..',
+	'J':'.---',
+	'K':'-.-',
+	'L':'.-..',
+	'M':'--',
+	'N':'-.',
+	'O':'---',
+	'P':'.--.',
+	'Q':'--.-',
+	'R':'.-.',
+	'S':'...',
+	'T':'-',
+	'U':'..-',
+	'V':'...-',
+	'W':'.--',
+	'X':'-..-',
+	'Y':'-.--',
+	'Z':'--..',
+	' ':' '
+	}
+
+i = 1
+j = 0
+talk = ""
+
+if (2 > len(sys.argv)):
+	talk = "<a-zA-Z string>"
+while (i < len(sys.argv)):
```

Comment:
Why use index in Python 😱 it's not C, you don't need to iterate through your array or string with an index, you can use `for` as you did it below